app/routes/main/Main.js

```diff
@@ -2,32 +2,32 @@ import React from "react";
 import { TabNavigator } from "react-navigation";
 import { Icon } from "react-native-elements";
 
-import Details from "./athleteDetails/AthleteDetails";
+import AthleteDetails from "./athleteDetails/AthleteDetails";
 import ClubFeed from "./clubFeed/ClubFeed";
-import PerformanceMeter from "./athletePerformanceLevel/AthletePerformanceLevel";
-import RacePredictor from "./clubEvents/ClubEvents";
+import AthletePerformanceLevel from "./athletePerformanceLevel/AthletePerformanceLevel";
+import ClubEvents from "./clubEvents/ClubEvents";
 import theme from "../../theme/theme";
 
 const Main = TabNavigator(
   {
-    PerformanceMeter: {
-      screen: PerformanceMeter,
+    AthletePerformanceLevel: {
```

Comment:
may be we should write here
[AthletePerformanceLevel.name] : {

Reply:
I don't get your point!